src/control/server/harness.go

```diff
@@ -37,27 +38,24 @@ import (
 // IOServerHarness is responsible for managing IOServer instances
 type IOServerHarness struct {
 	sync.RWMutex
-	log       logging.Logger
-	instances []*IOServerInstance
-	started   bool
-	errChan   chan error
+	log         logging.Logger
+	instances   []*IOServerInstance
+	started     uint32
+	restartable uint32
```

Comment:
This doesn't seem to be used for anything. Seems like "restartable" should be a calculated state rather than a boolean value.